<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1413</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="button_clear_dataset">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #4CAF50; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
border-radius:15px;
 background-color: white; 
  color: black; 
  border: 4px solid #4CAF50;
}
QPushButton:hover {

	background-color: rgb(174, 255, 137);
}</string>
    </property>
    <property name="text">
     <string>Delete Latest Dataset in AWS</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="grafik1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>1031</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_update_dataset">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #4CAF50; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
border-radius:15px;
 background-color: white; 
  color: black; 
  border: 4px solid #4CAF50;
}
QPushButton:hover {

	background-color: rgb(174, 255, 137);
}</string>
    </property>
    <property name="text">
     <string>Update Dataset Content in AWS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_update_csv">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #4CAF50; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
border-radius:15px;
 background-color: white; 
  color: black; 
  border: 4px solid #4CAF50;
}
QPushButton:hover {

	background-color: rgb(174, 255, 137);
}</string>
    </property>
    <property name="text">
     <string>Update and Save Dataset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_open_csv">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #4CAF50; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
border-radius:15px;
 background-color: white; 
  color: black; 
  border: 4px solid #4CAF50;
}
QPushButton:hover {

	background-color: rgb(174, 255, 137);
}</string>
    </property>
    <property name="text">
     <string>Open CSV File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_show_csv">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #4CAF50; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
border-radius:15px;
 background-color: white; 
  color: black; 
  border: 4px solid #4CAF50;
}
QPushButton:hover {

	background-color: rgb(174, 255, 137);
}</string>
    </property>
    <property name="text">
     <string>Print Dataset Content</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_show_lastgraphic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #4CAF50; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
border-radius:15px;
 background-color: white; 
  color: black; 
  border: 4px solid #4CAF50;
}
QPushButton:hover {

	background-color: rgb(174, 255, 137);
}</string>
    </property>
    <property name="text">
     <string>Display Last Created Graphic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_update_graphics">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #4CAF50; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
border-radius:15px;
 background-color: white; 
  color: black; 
  border: 4px solid #4CAF50;
}
QPushButton:hover {

	background-color: rgb(174, 255, 137);
}</string>
    </property>
    <property name="text">
     <string>Update CSV and Display Graphic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clearConsole">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>700</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #4CAF50; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
border-radius:15px;
 background-color: white; 
  color: black; 
  border: 4px solid #4CAF50;
}
QPushButton:hover {

	background-color: rgb(174, 255, 137);
}</string>
    </property>
    <property name="text">
     <string>Clear Console</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GRAPHICS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DATASET OPERATIONS</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="console_output">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>650</y>
      <width>1031</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbutton_time">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Indexed</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbutton_num">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Number Indexed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CONSOLE</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>1391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
